### ISSUE TITLE
Jar crashes

### ISSUE THREAD

**Issue by 8uurg** [lkroesen/BubbleBobble#54]
Error below:
```
Exception in thread "main" java.lang.IllegalArgumentException: URI is not hierarchical
        at java.io.File.<init>(Unknown Source)
        at website.frontrow.util.FileNameCollector.obtain(FileNameCollector.java:31)
        at website.frontrow.Launcher.main(Launcher.java:58)
```

Snippet where the exception occurred
```java
        URL url = this.getClass().getResource(dir);
        File levelDir = new File(url.toURI());
```

**Comment by lkroesen** [comment]
http://stackoverflow.com/questions/25690994/uri-is-not-hierarchical-when-loading-a-file-from-a-jar

**Comment by lkroesen** [comment]
Fixed in #56.
